src/websockets/framing.py

```diff
@@ -46,8 +61,24 @@
 # Using a list optimizes `code in EXTERNAL_CLOSE_CODES`.
 EXTERNAL_CLOSE_CODES = [1000, 1001, 1002, 1003, 1007, 1008, 1009, 1010, 1011]
 
-FrameData = collections.namedtuple(
-    "FrameData", ["fin", "opcode", "data", "rsv1", "rsv2", "rsv3"]
+
+Data = Union[str, bytes]
```

Comment:
One of the types that may need exporting.

Reply:
I moved it to `websockets.typing` and documented it.